addons/website/static/src/js/editor/snippets.options.js

```diff
@@ -1028,8 +1039,19 @@ options.registry.layout_column = options.Class.extend({
      * @see this.selectClass for parameters
      */
     selectCount: async function (previewMode, widgetValue, params) {
+        let $row = this.$('> .row:not(.o_we_zero_cols_content)');
+        if (!$row.length) {
+            this.$target.contents().wrapAll($('<div class="row"><div class="col-lg-12"/></div>'));
+            $row = this.$('> .row');
```

Comment:
Instead of wrapping in a row then searching the row, can't you create the row and wrap in it, avoid the search?
```javascript
$row = $('<div class="row"><div class="col-lg-12"/></div>');
this.$target.contents().wrapAll($row);
```
Mmh... maybe strange to read `wrapAll($row)`... but jquery is strange

Reply:
`wrapAll` makes a copy of the wrapping element, so in your code `$row` is never actually inserted into the DOM. I could do `$row = this.$target.contents().wrapAll($('<div class="row"><div class="col-lg-12"/></div>')).parent();` however, since `wrapAll` returns the wrapped elements. Not sure that's any better though.

Reply:
Keep your implementation then :+1: